apps/docs/app/ui/layout/wrapper/wrapper.css

```diff
@@ -1,20 +1,17 @@
 .hd-layout__wrapper {
-    width: min(100% - 2rem, var(--layout-width));
+    width: min(100% - 1rem, var(--layout-width));
     margin-inline: auto;
 }
 
 .hd-layout__fluid {
-    padding-inline: var(--hd-space-2);
+    padding-inline: var(--hd-space-1);
```

Comment:
Same thing, keep `var(--hd-space-2)` padding

Reply:
With 2 the layout break too soon, I kinda tried to stay on the 48 rem breakpoint, otherwise we'd have to go 48.75rem breakpoint which is odd.